pkg/action/unpack.go

```diff
@@ -19,7 +19,7 @@ type YamlMtree struct {
 }
 
 // excludes is a list of dirs that are excluded from the mtree check, usually those that are modified
-var excludes = []string{"var/cache/luet", "usr/local/tmp", "oem/", "usr/local/cloud-config", "usr/local/lost+found", "lost+found", "tmp/", "mnt/"}
+var excludes = []string{"var/cache/luet", "oem/", "usr/local/cloud-config", "usr/local/lost+found", "lost+found", "tmp/", "mnt/"}
```

Comment:
Shall we exclude `/usr/local` at all too?

Reply:
This list was basically made on trial and error while testing the different upgrades. 

The only modified parts that I found under `usr/local`  where the cloud-config and lost+found, which makes sense.

What other parts in there could be modified from build to deploy?

Im asking because I rather fine-grain this that skip a lot of dirs and miss something to check

Reply:
I think we never faced this because our tests are brittle. Until https://github.com/rancher-sandbox/cOS-toolkit/commit/f4d250f41006da171877a54573ada94a5108437d was merged, during upgrade you could have the oem partition and persistency mounted :facepalm:

Reply:
in that case, indeed we should skip the full dir!